packages/quip-cli/src/commands/init.ts

```diff
@@ -79,24 +85,32 @@ export default class Init extends Command {
         }),
         json: flags.boolean({
             char: "j",
-            description: "output responses in JSON (must provide --name)",
+            description:
+                "output responses in JSON (must provide --name and --id)",
```

Comment:
oclif provides a `dependsOn: ["name", "id"]` flag option for this

https://oclif.io/docs/flags